projects/1.16/assets/zombie-extreme/zombie-extreme/lang/zh_cn.json

```
@@ -0,0 +1,408 @@
+  "block.zombie_extreme.vendingmachine": "自动售货机",
+  "block.zombie_extreme.wire": "铁丝网片",
+  "block.zombie_extreme.wirefence": "铁丝网",
+  "effect.antiradon_protection": "防辐射",
+  "effect.infection": "感染",
+  "entity.zombie_extreme.bombsuitzombie": "防爆服僵尸",
+  "entity.zombie_extreme.boomerzombie": "呕吐僵尸",
+  "entity.zombie_extreme.clicker_zombie": "尖啸者",
+  "entity.zombie_extreme.crawler_zombie": "半身僵尸",
+  "entity.zombie_extreme.demolisher": "破坏者",
+  "entity.zombie_extreme.drawnedzombie": "腐败僵尸",
+  "entity.zombie_extreme.explosive_zombie": "爆炸僵尸",
+  "entity.zombie_extreme.fast_zombie_v_2": "迅捷感染者",
+  "entity.zombie_extreme.heavy_military": "重装部队",
+  "entity.zombie_extreme.infected_bear": "感染熊",
+  "entity.zombie_extreme.infected_dog": "感染狗",
+  "entity.zombie_extreme.infected_hazmat": "感染危险品",
+  "entity.zombie_extreme.infected_spec_ops_soldier": "感染特种部队士兵",
+  "entity.zombie_extreme.infectedperson": "感染者",
+  "entity.zombie_extreme.militarysoldier": "堕落军人",
+  "entity.zombie_extreme.murku_force_unit": "穆尔库部队",
+  "entity.zombie_extreme.night_hunter": "夜魔",
+  "entity.zombie_extreme.parasite": "寄生虫",
+  "entity.zombie_extreme.poisonous_zombie": "猛毒僵尸",
+  "entity.zombie_extreme.policezombie": "感染者警察",
+  "entity.zombie_extreme.radioactive_zombie": "放射性僵尸",
+  "entity.zombie_extreme.rat_king": "鼠王",
+  "entity.zombie_extreme.revived": "复活者",
+  "entity.zombie_extreme.soldier_zombie": "士兵僵尸",
+  "entity.zombie_extreme.survial": "伤员",
+  "entity.zombie_extreme.survival": "幸存者",
+  "entity.zombie_extreme.wastland_spider": "荒地蜘蛛",
+  "entity.zombie_extreme.zero_patient": "零号病人",
+  "entity.zombie_extreme.zombie_brute": "僵尸野蛮人",
+  "item.makarovempty.name": "未装弹的马卡洛夫手枪",
+  "item.minecraft.lingering_potion.effect.antiradon_protection": "持久性抗辐射保护",
+  "item.minecraft.lingering_potion.effect.bleeding": "持续出血",
+  "item.minecraft.lingering_potion.effect.infection": "持续感染物品",
+  "item.minecraft.lingering_potion.effect.radiation": "持续性辐射",
+  "item.minecraft.potion.effect.antiradon_protection": "反辐射防护",
+  "item.minecraft.potion.effect.bleeding": "流血",
+  "item.minecraft.potion.effect.infection": "感染",
+  "item.minecraft.potion.effect.radiation": "辐射",
+  "item.minecraft.splash_potion.effect.antiradon_protection": "喷溅型防辐射药水",
+  "item.minecraft.splash_potion.effect.bleeding": "大出血",
+  "item.minecraft.splash_potion.effect.infection": "急性感染",
+  "item.minecraft.splash_potion.effect.radiation": "急性辐射症",
+  "item.minecraft.tipped_arrow.effect.antiradon_protection": "反辐射保护之箭",
+  "item.minecraft.tipped_arrow.effect.bleeding": "流血之箭",
+  "item.minecraft.tipped_arrow.effect.radiation": "辐射之箭",
+  "item.p_2_2_8.name": "P228",
+  "item.zombie_extreme.accumulator": "累加器",
+  "item.zombie_extreme.adrenalin": "肾上腺素（提供36s的力量III跟抗性提升III）",
+  "item.zombie_extreme.ammo_p_2_2_8": "P228弹匣",
+  "item.zombie_extreme.ammomakarov": "马卡洛夫弹匣",
+  "item.zombie_extreme.ancient_flesh": "远古血肉",
+  "item.zombie_extreme.anti_selective_vaccine": "血清（治疗中毒及其他症状）",
```

```suggestion
  "item.zombie_extreme.anti_selective_vaccine": "万用疫苗",
```
原文如此（）
并未找到原文对应的术语，仅建议，不过血清应该是Serum